concepticondata/conceptlists/Strong-1911-150.tsv

```diff
@@ -0,0 +1,151 @@
+Strong-1911-150-3	3	Arm	1673	ARM
+Strong-1911-150-4	4	Arrow	977	ARROW
+Strong-1911-150-5	5	Ashes	646	ASH
+Strong-1911-150-6	6	Bad	1292	BAD
+Strong-1911-150-7	7	Bamboo	1927	BAMBOO
+Strong-1911-150-8	8	Banana	868	BANANA
+Strong-1911-150-9	9	Barter	370	BARTER
+Strong-1911-150-10	10	Belly	1251	BELLY
+Strong-1911-150-11	11	Bird	937	BIRD
+Strong-1911-150-12	12	Bite	1403	BITE
+Strong-1911-150-13	13	Black	163	BLACK
+Strong-1911-150-14	14	Blood	946	BLOOD
+Strong-1911-150-15	15	Boat	1844	BOAT
+Strong-1911-150-16	16	Bone	1394	BONE
+Strong-1911-150-17	17	Bow (noun)	994	BOW
+Strong-1911-150-18	18	Branch	1531	BRANCH
+Strong-1911-150-19	19	Bring	692	BRING
+Strong-1911-150-20	20	Bury	1719	BURY
+Strong-1911-150-21	21	Butterfly	1791	BUTTERFLY
+Strong-1911-150-22	22	Centipede	1974	CENTIPEDE
+Strong-1911-150-23	23	Chest	1592	CHEST
+Strong-1911-150-24	24	Charcoal	1188	CHARCOAL
+Strong-1911-150-25	25	Child	1304	CHILD (YOUNG HUMAN)
+Strong-1911-150-26	26	Claw (of bird)	72	CLAW
+Strong-1911-150-27	27	Cloud	1489	CLOUD
+Strong-1911-150-28	28	Club	1763	CLUB
+Strong-1911-150-29	29	Cocoanut	970	COCONUT
+Strong-1911-150-30	30	Come	1446	COME
+Strong-1911-150-31	31	Crocodile	1857	CROCODILE
+Strong-1911-150-32	32	Darkness	11	DARKNESS
+Strong-1911-150-33	33	Die	1494	DIE
+Strong-1911-150-34	34	Digging stick	339	DIGGING STICK
+Strong-1911-150-35	35	Dog	2009	DOG
+Strong-1911-150-36	36	Drink	1401	DRINK
+Strong-1911-150-37	37	Ear	1247	EAR
+Strong-1911-150-38	38	Earth (ground)	2159	GROUND
+Strong-1911-150-39	39	Eat	1336	EAT
+Strong-1911-150-40	40	Egg	744	EGG
+Strong-1911-150-41	41	Elbow	981	ELBOW
+Strong-1911-150-42	42	Eye	1248	EYE
+Strong-1911-150-43	43	Face	1560	FACE
+Strong-1911-150-44	44	Far off	1406	FAR
+Strong-1911-150-45	45	Father	1217	FATHER
+Strong-1911-150-46	46	Feather	1201	FEATHER
+Strong-1911-150-47	47	Finger	1303	FINGER
+Strong-1911-150-48	48	Fire	221	FIRE
+Strong-1911-150-49	49	Fish	227	FISH
+Strong-1911-150-50	50	Flesh	2259	FLESH
+Strong-1911-150-51	51	Flower	239	FLOWER
+Strong-1911-150-52	52	Fly (noun)	1504	FLY (INSECT)
+Strong-1911-150-53	53	Foot	1301	FOOT
+Strong-1911-150-54	54	Forest	420	FOREST
+Strong-1911-150-55	55	Fowl	265	FOWL
+Strong-1911-150-56	56	Fruit	1507	FRUIT
+Strong-1911-150-57	57	Garden	586	GARDEN
+Strong-1911-150-58	58	Ghost	1175	GHOST
+Strong-1911-150-59	59	Give	1447	GIVE
+Strong-1911-150-60	60	Go	695	GO
+Strong-1911-150-61	61	Good	1035	GOOD
+Strong-1911-150-62	62	Great	1202	BIG
+Strong-1911-150-63	63	Hair (of head)	2648	HAIR (HEAD)
+Strong-1911-150-64	64	Hand	1277	HAND
+Strong-1911-150-65	65	Hard	1884	HARD
+Strong-1911-150-66	66	Head	1256	HEAD
+Strong-1911-150-67	67	Hear	1408	HEAR
+Strong-1911-150-68	68	Hill	614	HILL
+Strong-1911-150-69	69	Hook	1071	HOOK
+Strong-1911-150-70	70	Hot	1286	HOT
+Strong-1911-150-71	71	House	1252	HOUSE
+Strong-1911-150-72	72	Husband	1200	HUSBAND
+Strong-1911-150-73	73	Kangaroo	164	KANGAROO
+Strong-1911-150-74	74	Know	3626	KNOW
+Strong-1911-150-75	75	Leaf	628	LEAF
+Strong-1911-150-76	76	Log	1803	WOOD
```

Comment:
There's quite a number of lists that have *log* and most of them are unmapped. I think here I'd rather opt for introducing a new concept (and mapping the other lists) than mapping to `WOOD`.

Reply:
ok, i'll unmap.